velox/docs/develop/scalar-functions.rst

```diff
@@ -580,15 +580,15 @@ The “rows” parameter specifies the set of rows in the incoming batch to
 process. This set may not include all the rows. By default, a vector function
 is assumed to have the default null behavior, e.g. null in any input produces
 a null result. In this case, the expression evaluation engine will exclude
-rows with nulls from the “rows” specified in the call to “apply”. If a
-function has a different behavior for null inputs, it must override the
-isDefaultNullBehavior method to return false.
+rows with nulls from the “rows” specified in the call to “apply”. If a function
+has a different behavior for null inputs, metadata flag must be provided with
```

Comment:
> metadata flag must be provided with defaultNullBehavior set to false during registration

This is a lot of words, but the reader still need to read the Registration section to understand what to do. Hence, I suggest to shorten to

`If a function has a different behavior for null inputs it must specify that during registration. See ...`

Let's also make "Registration" a link.